api.planx.uk/modules/send/idox/nexus.ts

```diff
@@ -68,30 +68,35 @@ export async function sendToIdoxNexus(
 
   // localAuthority is only parsed for audit record, not client-specific
   const localAuthority = req.params.localAuthority;
-  const uniformClient = getUniformClient();
+  const idoxNexusClient = getIdoxNexusClient();
 
   // confirm that this session has not already been successfully submitted before proceeding
   const submittedApp = await checkUniformAuditTable(payload?.sessionId);
-  const isAlreadySubmitted =
+  const _isAlreadySubmitted =
     submittedApp?.submissionStatus === "PENDING" && submittedApp?.canDownload;
-  if (isAlreadySubmitted) {
-    return res.status(200).send({
-      sessionId: payload?.sessionId,
-      idoxSubmissionId: submittedApp?.submissionId,
-      message: `Skipping send, already successfully submitted`,
-    });
-  }
+  // if (isAlreadySubmitted) {
+  //   return res.status(200).send({
+  //     sessionId: payload?.sessionId,
+  //     idoxSubmissionId: submittedApp?.submissionId,
+  //     message: `Skipping send, already successfully submitted`,
+  //   });
+  // }
 
   try {
     // Request 1/4 - Authenticate
-    const { token, organisation, organisationId } =
-      await authenticate(uniformClient);
+    const { token, organisations } = await authenticate(idoxNexusClient);
+
+    // TEMP - Mock organisations do NOT correspond to council envs, so randomly alternate submissions among ones we have access to for initial testing
+    //   Switch to `team_integrations`-based approach later
+    const orgIds = Object.keys(organisations);
+    const randomOrgId = orgIds[Math.floor(Math.random() * orgIds.length)];
+    const randomOrg = organisations[randomOrgId];
```

Comment:
This is a bit of an odd one! Basically - I want to be able to share a single testing link _per service_ with Idox, not _per council & per service_ because that's 4x the flow content to adjust - so this is an easy solution for now.

Any flow in any team using "Idox Nexus" Send destination will submit to one of the four random orgs we have access to ! Once we get "real" credentials, we'll store them in `team_integrations` and do a lookup there